frontend/src/actions/prefAndProfileActions.tsx

```diff
@@ -67,6 +67,12 @@ export interface IUpdateNicknameActionPayload {
   nickname: string
 }
 
+export interface IUpdatePasswordActionPayload {
+  old_password: string
```

Comment:
let's use camelCase - convert to snake_case inside api client method

Reply:
Done with one ts-ignore line, by now not understand how to correct it. converting works.